### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by rsedykh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,207 @@
...
+[![](https://readthedocs.org/projects/pyuploadcare/badge/?version=latest)](https://pyuploadcare.readthedocs.io/)
+[![](https://img.shields.io/badge/tech-stack-0690fa.svg?style=flat)](https://stackshare.io/uploadcare)
+
+Build file handling in minutes. Upload or accept UGC, store, transform, optimize, and deliver images, videos, and documents to billions of users.
+
+* [Features](#features)
+* [Documentation](#documentation)
+* [Requirements and installation](#requirements-and-installation)
+* [Usage example](#usage-example)
+* [Use cases](#use-cases)
```
Comment (evgkirov) [inline README.md:32]: Broken link (missing content).

File: README.md [hunk 2]
```diff
@@ -1,51 +1,207 @@
...
+
+| Py/Dj | 2.2 | 3.0 | 3.1 | 3.2 | 4.0 | 4.1 | 4.2 | 5.0 |
+| ----- | --- | --- | --- | --- | --- | --- | --- | --- |
+| 3.8   | X   | X   | X   | X   | X   | X   | X   |     |
+| 3.9   | X   | X   | X   | X   | X   | X   | X   |     |
+| 3.10  |     |     |     | X   | X   | X   | X   | X   |
+| 3.11  |     |     |     |     |     | X   | X   | X   |
+| 3.12  |     |     |     |     |     |     | X   | X   |
+
+In order to install `pyuploadcare`, run these commands in CLI:
```
Comment (evgkirov) [inline README.md:107]: It says "commands", but there's actually just one command (I know this piece is copied from RTD).

File: README.md [hunk 3]
```diff
@@ -1,51 +1,207 @@
...
+
+```bash
+pip install pyuploadcare
+```
+
+or, if you prefer it the old way:
+
+```bash
+easy_install pyuploadcare
```
Comment (evgkirov) [inline README.md:116]: `easy_install` is so dead; I doubt if anyone uses it in 2024 (I know this piece is copied from RTD).

File: README.md [hunk 4]
```diff
@@ -1,51 +1,207 @@
...
+```bash
+pip install pyuploadcare[django]
+```
+
+## Usage example
+
+<!-- TODO: The idea of this example is to show how upload and deliver an image -->
+
+<!-- TODO: Update it with "and deliver an image" -->
+Let's add [File Uploader](https://uploadcare.com/docs/file-uploader/) to an existing Django project.
```
Comment (evgkirov) [inline README.md:130]: I feel like we should start with an example demonstrating the general use of the library before introducing the Django example. For instance, we could showcase a typical upload API request and response, similar to the example found here: https://uploadcare.com/docs/uploading-files/#upload-api-request-and-response

File: README.md [hunk 5]
```diff
@@ -63,16 +219,92 @@ You can find an example project [here](https://github.com/uploadcare/pyuploadcar
...
-## Documentation
+## Testing
+
+To run tests using [Github Actions](https://github.com/uploadcare/pyuploadcare/actions) workflows, but locally, install the [act](https://github.com/nektos/act) utility, and then run it:
+
+```make test_with_github_actions```
+
+This runs the full suite of tests across Python and Django versions.
+
+<!--
```
Comment (evgkirov) [inline README.md:234]: Work in progress?

File: README.md [hunk 6]
```diff
@@ -63,16 +219,92 @@ You can find an example project [here](https://github.com/uploadcare/pyuploadcar
...
-Detailed documentation is available [on RTD](https://pyuploadcare.readthedocs.io/en/latest/).
+We've developed a demo app that showcases most of the features. You can install [pyuploadcare-example](https://github.com/uploadcare/pyuploadcare-example) using Docker or without it. You can use it as a reference or even base your project on it.
 
-## Feedback
+## Suggestions and questions
 
-Issues and PRs are welcome. You can provide your feedback or drop us a support
-request at [hello@uploadcare.com](hello@uploadcare.com).
+[Contributing guide](https://github.com/uploadcare/.github/blob/master/CONTRIBUTING.md)  
+[Security policy](https://github.com/uploadcare/uploadcare-swift/security/policy)
```
Comment (evgkirov) [inline README.md:309]: We don't do "swift" in here.

File: README.md (hunk 4 above)
Comment (rsedykh) [reply README.md:130]: Agree. Can you please update it? And I think let's have two:
- simple upload via API request
- Django magic with File Uploader

WDYT?

File: README.md (hunk 5 above)
Comment (rsedykh) [reply README.md:234]: Or it shouldn't belong here. IDK.. Maybe it's better as a separate section in actual documentation.

File: README.md (hunk 6 above)
Comment (rsedykh) [reply README.md:309]: Oops.

File: README.md [hunk 7]
```diff
@@ -1,78 +1,204 @@
...
+{% for photo in photos %}
+    <h2>{{ photo.title }}</h2>
+    <img src="{{ photo.photo.cdn_url }}-/resize/400x300/-/effect/flip/-/effect/grayscale/">
+{% endfor %}
+```
+
+<!-- TODO: Maybe we should link to the URL builder pyuploadcare docs? -->
+(Refer to Uploadcare [image processing docs](https://uploadcare.com/docs/transformations/image/) for more information).
+
+<!-- TODO: Connect this example to the rest of them -->
```
Comment (evgkirov) [inline README.md:163]: It doesn't really demonstrate anything new here (apart from the widget configuration — but that could be a part of the docs).

File: README.md (hunk 4 above)
Comment (evgkirov) [reply README.md:131]: Added

Comment (evgkirov) [reply README.md:131]: f572a9a7782a24ef73a30b3d88d6f2cf8de4caca

File: README.md [hunk 8]
```diff
@@ -78,11 +78,50 @@ To use in Django project install with extra dependencies:
...
+    ucare_file = uploadcare.upload(file_object)
+```
+
+And your file is now uploaded to the Uploadcare CDN. But how do you access it from the web? It’s really simple:
+
+```python
+print(ucare_file.cdn_url)  # file URL, e.g.: https://ucarecdn.com/7d4e9387-26c2-4e58-840e-9254f1871c94/
+```
+
+If you don’t have the “Automatic file storing” setting enabled in your project, don’t forget to store the uploaded file:
```
Comment (rsedykh) [inline README.md:105]: We've changed defaults for new users (now default is auto, and in dashboard autostore is on), so I guess we can skip this.

Comment (rsedykh) [inline README.md:105]: @evgkirov We've changed defaults for new users (now default is auto, and in dashboard autostore is on), so I guess we can skip this.

File: README.md [hunk 9]
```diff
@@ -1,78 +1,248 @@
...
 
-![](https://ucarecdn.com/f0894ef2-352e-406a-8279-737dd6e1f10c/-/resize/800/manual_crop.png)
+There’s a lot more to uncover. For more information please refer to the [documentation](#documentation).
 
-## Documentation
+### Django integration
+
+<!-- TODO: The idea of this example is to show how upload and deliver an image -->
+
+<!-- TODO: Update it with "and deliver an image" -->
```
Comment (evgkirov) [inline README.md:160]: I've added a paragraph into the intro below. Does it serve this purpose?

File: README.md [hunk 10]
```diff
@@ -1,78 +1,248 @@
...
+print(ucare_file.cdn_url)  # https://ucarecdn.com/640fe4b7-7352-42ca-8d87-0e4387957157/-/scale_crop/512x512/smart/
 ```
 
-![](https://ucarecdn.com/f0894ef2-352e-406a-8279-737dd6e1f10c/-/resize/800/manual_crop.png)
+There’s a lot more to uncover. For more information please refer to the [documentation](#documentation).
 
-## Documentation
+### Django integration
+
+<!-- TODO: The idea of this example is to show how upload and deliver an image -->
```
Comment (evgkirov) [inline README.md:158]: What should we do with this *TODO*?

Comment (rsedykh) [reply README.md:158]: Anything. Example is very much up to you. You know our products well enough. :-)

File: README.md (hunk 9 above)
Comment (rsedykh) [reply README.md:160]: 🤩

**Review by evgkirov — APPROVED** [review]

**Pull request merged**